conformity/error.py

```diff
@@ -3,10 +3,16 @@
 import attr
 
 
+ERROR_CODE_INVALID = 'INVALID'
```

Comment:
Conformity uses double quotes for strings.

Reply:
Fixed.